operator/hack/common.sh

```diff
@@ -80,10 +84,10 @@ function retry() {
 }
 
 function check_version_tag() {
-  local -r csv_version="$1"
+  local -r starting_csv_version="$1"
   local -r allow_dirty_tag="$2"
 
-  if [[ "$csv_version" == *-dirty ]]; then
+  if [[ "$starting_csv_version" == *-dirty ]]; then
```

Comment:
Sorry, I did not mean that and you took my suggestion a bit too far. I meant to rename `csv_version` to `starting_csv_version` in `apply_operator_manifests` (and its call chain with `operator/hack/olm-operator-install.sh`) but not in other functions of `operator/hack/common.sh`.

It makes sense to keep calling it `csv_version` in `check_version_tag`, `approve_install_plan` and so on.

Reply:
It seems my IDE does not know about `local` and renames all occurrences in the file. Let's keep it as is.

Reply:
I added this comment because I believe renaming in other functions makes things more confusing. I did not expect that keeping it as-is would be a good option.

Reply:
OK, done.